src/utils/Reset.scss

```diff
@@ -0,0 +1,10 @@
+body,
+h1,
+input {
+  margin: 0;
```

Comment:
Consider using more specific selectors instead of global selectors like `body`, `h1`, and `input`. This can help avoid unintended side effects. 🤔